.github/workflows/ci.yml

```diff
@@ -36,7 +36,12 @@ jobs:
         run: sbt compile Test/compile
 
       - name: Run tests
-        run: sbt test
+        run: |
+          sbt -scala-version 2.12.14 test
```

Comment:
Hmmm actually... this fails for me locally, does it work on your machine? 🤔 

```
$ sbt -scala-version 2.13.6 test
[info] welcome to sbt 1.5.5 (Red Hat, Inc. Java 11.0.13)
[info] loading settings for project global-plugins from plugins.sbt ...
[info] loading global plugins from /home/plipski/.sbt/1.0/plugins
[info] loading settings for project akka-serialization-helper-build-build from plugins.sbt ...
[info] loading project definition from /home/plipski/akka-serialization-helper/project/project
[info] loading settings for project akka-serialization-helper-build from build.sbt,plugins.sbt ...
[info] loading project definition from /home/plipski/akka-serialization-helper/project
[info] loading settings for project akka-serialization-helper from build.sbt ...
[info] resolving key references (13734 settings) ...
[info] set current project to akka-serialization-helper (in build file:/home/plipski/akka-serialization-helper/)
[error] Expected symbol
[error] Not a valid command: -
[error] Expected end of input.
[error] Expected '--'
[error] Expected 'debug'
[error] Expected 'info'
[error] Expected 'warn'
[error] Expected 'error'
[error] Expected 'addPluginSbtFile'
[error] -scala-version
[error]  ^
```

Reply:
No, it doesn't. I looked into CI logs and saw that we use `olafurpg/setup-scala@v10` which has additional args for sbt and one of those is `-scala-version`. Under the hood, it's `++ version` which is a regular sbt parameter for the version. So I used that for defining version.

Reply:
Hmmm okay so could you change the syntax here for CI invocation so that it's portable to our local machines? or is `-scala-version` significantly distinct from `++ version`?

Reply:
Sure. It's the same but `-scala-version <version>` is more readable so that's why I picked it over `++ <version>`.

Reply:
Huh okay so at least add a comment how to reproduce the same test run locally (it's not gonna be obvious in a few months)... or is there a way to configure our local `sbt` so that `-scala-version` option is accepted as it is now in the CI?

Reply:
Changed to `++ <version>` because we can run it locally.